asu/routers/api.py

```diff
@@ -271,3 +271,15 @@ def api_v1_build_post(
     response.status_code = status
 
     return content
+
+
+@router.get("/builder_stats")
+def api_v1_builder_stats():
```

Comment:
Let's call this just api/v1/stats and I'll merge it?

Reply:
Will do.